runtimes/eden/src/lib.rs

```diff
@@ -403,8 +426,31 @@ sp_api::impl_runtime_apis! {
 			// We did not include the offences and sessions benchmarks as they are parity
 			// specific and were causing some issues at compile time as they depend on the
 			// presence of the staking and elections pallets.
+			use cumulus_primitives_core::{Fungibility::Fungible, MultiAsset, MultiLocation, Parent};
+			use frame_benchmarking::{Benchmarking, BenchmarkBatch,BenchmarkError};
+
+			use crate::constants::EXISTENTIAL_DEPOSIT;
+
+			impl pallet_xcm::benchmarking::Config for Runtime {
+				fn reachable_dest() -> Option<MultiLocation> {
+					Some(Parent.into())
+				}
 
-			use frame_benchmarking::{Benchmarking, BenchmarkBatch,BenchmarkError, TrackedStorageKey, add_benchmark};
+				fn teleportable_asset_and_dest() -> Option<(MultiAsset, MultiLocation)> {
+					// Relay/native token can be teleported between People and Relay.
+					Some((
+						MultiAsset {
+							fun: Fungible(EXISTENTIAL_DEPOSIT),
```

Comment:
Applying NODL's existential deposit to a parent asset is not reliable. Can we use the existential deposit definition from polkadot primitives?

Reply:
Line 430

Reply:
Using a new constant for parent Existential Deposit